M2/Macaulay2/m2/last.m2

```diff
@@ -58,11 +55,7 @@ addStartFunction( () -> (
 addStartFunction( () -> (
 	  if not nobanner then (
 	       if topLevelMode === TeXmacs then stderr << TeXmacsBegin << "verbatim:";
-	       relevant := select(loadedPackages,mentionQ);
-	       if #relevant > 0 then (
-	       	    hd := "with packages: ";
-	       	    stderr << hd << wrap(printWidth-#hd, concatenate between_", " sort apply(relevant,toString)) << endl;
-		    );
+	       stderr << "Need info? Try 'help', 'copyright', or 'cite'." << endl;
```

Comment:
I think the question is unnecessary.

Reply:
So maybe just `Try 'help', 'copyright', or 'cite'.`?